.github/workflows/main.yml

```diff
@@ -85,6 +69,13 @@ jobs:
           mkdir -p '${{ runner.tool_cache }}/ccache'
           echo 'CCACHE_DIR=${{ runner.tool_cache }}/ccache' >> $GITHUB_ENV
         shell: bash
+      # Remove and reinstall rust as default install seems broken on Ubuntu arm
+      - run: |
+            rm -rf $HOME/.rustup $HOME/.cargo
```

Comment:
No need for the `|` here if the command is just one line.  You can do `run: rm -rf $HOME/.rustup $HOME/.cargo` on the same line I think.